app/src/main/java/org/xwiki/android/sync/auth/AuthenticatorActivity.kt

```diff
@@ -371,6 +377,21 @@ class AuthenticatorActivity : AccountAuthenticatorActivity() {
         mAccountManager.setUserData(account, AccountManager.KEY_PASSWORD, accountPassword)
         mAccountManager.setUserData(account, PARAM_USER_SERVER, accountServer)
 
+        scope.launch {
+            val user = User("$accountName@$accountServer",
```

Comment:
Code style convention require here to use new line for first argument when it is multiline usage. Besides, for `$accountName@$accountServer` must be created some function, which will create this string. Better way here - to generate it inside of User model.

Reply:
Done.